[RadioMelodieBridge] Add Timestamp support and content fix

sysadminstory opened the pull request:
- Timestamp is set for every RSS item
- Remove ugly Faceook Javascript
- Rewrite relative links to absolute links

sysadminstory committed:
[RadioMelodieBridge] Add Timestamp support and content fix

- Timestamp is set for every RSS item
- Remove ugly Faceook Javascript
- Rewrite relative links to absolute links

em92 commented on bridges/RadioMelodieBridge.php:
```diff
@@ -13,6 +13,17 @@ public function collectData(){
 		$list = $html->find('div[class=displayList]', 0)->children();
+
+		$dateFormat = 'EEEE  d LLLL yyyy à HH:mm';
+		$dateFormatter = new IntlDateFormatter(
```
That would require intl extention to be enabled, which is not in minimal requirements of rss-bridge.
Can it be parsed via https://www.php.net/manual/ru/function.date-parse-from-format.php?

sysadminstory replied:
Hello !

As date_parse_from_format could not handle localized day of week and month names, I switched to strptime which is totally native (but needs to have access to the French locale)/

sysadminstory committed:
[RadioMelodieBridge] Switch date parsing from Intl to native PHP functions

'Intl' is not a requirements of RSS-Bridge, so I switched from 'Intl'
functions to native PHP functions to parse the item date.

sysadminstory committed:
[RadioMelodieBridge] Coding policy fix

Coding policy fix

em92 commented:
gj!
